tests/test_summary.py

```diff
@@ -0,0 +1,29 @@
+from airbase.summary.db import DB
+
+
+def test_countries():
+    countries = DB.countries()
+    assert isinstance(countries, list)
+    assert countries
+    assert all(isinstance(country, str) for country in countries)
```

Comment:
could maybe `assert` that a few country codes are in the list for consistency with the other tests here

Reply:
done